client/gutenberg/extensions/tiled-gallery/tiled-grid/constrained-array-rounding/index.js

```diff
@@ -64,12 +64,6 @@ const adjust_constrained_array = ( bound_array_int, adjustment ) => {
  * @returns {Array} Array with numeric elements rounded to integers, while their sum is preserved.
  */
 export const getRoundedConstrainedArray = ( bound_array, sum_value = false ) => {
-	// Convert associative arrays before working with them and convert them
-	// back before returning the values
-	// const keys = Object.keys( bound_array );
-
-	bound_array = Object.values( bound_array );
-
 	let bound_array_int = get_int_floor_array( bound_array );
 
 	const lower_sum = sum( map( bound_array_int, property( 'floor' ) ) );
```

Comment:
I think with lodash we can write `map( arr, property( 'floor' ) )` can be written `map( arr, 'floor' )`

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/29548/commits/3d4c72d49cb5ca88d3e18e3416d64951fbc31736